src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -144,7 +144,7 @@ private ExpressionSyntax ParseInterpolatedStringToken()
             }
 
             Debug.Assert(originalToken.ToFullString() == result.ToFullString()); // yield from text equals yield from node
-            return CheckFeatureAvailability(result, MessageID.IDS_FeatureInterpolatedStrings);
```

Comment:
this is an example of reverse-smear.  Three syntactic checks becomes one semantic one.